Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,33 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_environment",
+            category=['society'],
+            prompt="The environment",
+            random=True
+        )
+    )
+ 
+label monika_environment:
+    m 2eub "[player], do you know why I care about the environment?"
+    m 1esa "Well, there are a few reasons."
+    m 3euu "For one, I think it's beautiful."
```

Comment:
![1](https://user-images.githubusercontent.com/45746485/93007316-ffc8ae80-f567-11ea-858b-e26c6a100aae.png)
The mouth here could be changed I think, not a big deal. Suggestion:

![image](https://user-images.githubusercontent.com/45746485/93007334-3bfc0f00-f568-11ea-8838-eae07ab995b3.png)

Reply:
My only issue with this one is that it's similar to her first expression (2eub vs 3eub) and I want to avoid having the same/similar expressions if I can help it since that would feel artificial IMO. I'll see what I can do with this one.

Reply:
Do that, we can always find other expressions to try out and find the best one.